client/src/util/utmTag.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+/**
+ * Returns the URL tagged with application specific UTM tags.
+ *
+ * Per default this will override existing UTM tags, to ensure consistent tagging.
+ *
+ * @param { string } urlString - The original URL
+ * @param { { campaign?: string } } tags
+ * @param { boolean } [override=true]
+ *
+ * @returns {string} The URL with UTM parameters.
+ */
+export function utmTag(urlString, tags = {}, override = true) {
```

Comment:
Do we ever want to not override? I don't see any usage or tests for that (YAGNI)